.github/workflows/codeql-analysis.yml

```diff
@@ -17,7 +17,8 @@ on:
 jobs:
   analyze:
     name: Analyze
-    runs-on: windows-latest
+    # Move to Windows 2022 when CodeQL is supported there
+    runs-on: windows-2019
```

Comment:
I was also going to make this change earlier https://github.com/microsoft/Omex/pull/447 as I thought it was blocking changes :) if we make this change, would the plan be to move back to windows-latest at a later stage?

Reply:
According to this (https://github.com/actions/virtual-environments/issues/4856) server 2022 is being rolled out. As a naturally lazy engineer, I'd just wait, if it's not blocking.

Reply:
Yes, server 2022 is being rolled out as the windows-latest target, which is why this change is needed.
CodeQL doesn't support server 2022 - this is explicitly called out in that issue.
We should change it to 2019 and log an issue to track changing it back to latest when https://github.com/github/codeql-action/issues/850 is fixed.

Reply:
Note it is currently blocking package updates - https://github.com/microsoft/Omex/pull/444

Reply:
Ah, I've misread the error message completely, thanks for noticing this.